packages/optimize/.babelrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
The React Native app should have a similar root `babelrc`. So I have removed `babelrc` in other extensions like the core extension: https://github.com/adobe/aepsdk-react-native/tree/main/packages/core.
Do we have a reason to still keep this file?